docs/Building.md

```diff
@@ -0,0 +1,230 @@
+# Building `ros2_rust` packages
+
+This is a more detailed guide on how to build ROS 2 packages written in Rust that expands on the minimal steps in the README.
+
+In this guide, the Foxy distribution of ROS 2 is used, but newer distributions can be used by simply replacing 'foxy' with the distribution name everywhere.
+
+
+## Environment setup
+
+Building `rclrs` requires a standard [ROS 2 installation](https://docs.ros.org/en/foxy/Installation.html), and a few extensions.
+These extensions are: `colcon-cargo`, `colcon-ros-cargo`, `cargo-ament-build`. The former two are `colcon` plugins, and the latter is a `cargo` plugin.
```

Comment:
```suggestion
These extensions are: `colcon-cargo`, `colcon-ros-cargo`, `cargo-ament-build`. The first two are `colcon` plugins, and the third is a `cargo` plugin.
```
English is odd. Nothing's technically wrong with how you have it, but I think this looks a bit more "natural". Feel free to ignore, though.